src/pages/ImportPage.tsx

```diff
@@ -0,0 +1,89 @@
+import { Project } from "@microbit/makecode-embed/react";
+import { useEffect } from "react";
+import { IntlShape, useIntl } from "react-intl";
+import { useNavigate } from "react-router";
+import { useSearchParams } from "react-router-dom";
+// TODO: Use theme activitiesBaseUrl
+// import { useDeployment } from "../deployment";
+import { MicrobitOrgResource } from "../model";
+import { useStore } from "../store";
+import { createDataSamplesPageUrl } from "../urls";
+
+const ImportPage = () => {
+  const navigate = useNavigate();
+  const intl = useIntl();
+  // TODO: Use theme activitiesBaseUrl
+  // const { activitiesBaseUrl } = useDeployment();
+  const activitiesBaseUrl =
+    "http://open-in-createai.next-review.microbit.org.s3-website-eu-west-1.amazonaws.com/classroom/activities";
```

Comment:
TODO: Use theme activitiesBaseUrl
```suggestion
  const { activitiesBaseUrl } = useDeployment();
```